.circleci/config.yml

```diff
@@ -119,7 +119,11 @@ commands:
             make benchmarks-basic-build NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
             ccache -s
 
-  run-benchmarks:
+  run-and-or-upload-benchmarks:
```

Comment:
maybe the api here could be providing two jobs:

- run-benchmarks
- upload-benchmarks-results

and you would handle the shared steps within these jobs?

Reply:
Yeah, this API is not the best. We've been working on [a better way](https://github.com/conbench/conbench/tree/main/benchadapt) to separate the concerns of uploading benchmarks from running them. I'll try to separate this out better.